runtime/plaid/src/functions/api.rs

```diff
@@ -339,6 +339,14 @@ impl_new_sub_module_function_with_error_buffer!(
 #[cfg(feature = "aws")]
 impl_new_sub_module_function_with_error_buffer!(aws, kms, get_public_key, ALLOW_IN_TEST_MODE);
 
+// AWS ECR functions
+#[cfg(feature = "aws")]
+impl_new_function_with_error_buffer!(aws, ecr_list_repositories, ALLOW_IN_TEST_MODE);
```

Comment:
These should use `impl_new_sub_module_function_with_error_buffer` with ecr as the submodule